app/src/main/java/com/android/solvit/seeker/ui/provider/ProviderInfo.kt

```diff
@@ -201,19 +209,20 @@ fun ProviderPackages(provider: Provider, packages: List<PackageProposal>) {
         // Horizontal scrollable list
         LazyRow(
             modifier = Modifier.fillMaxWidth().testTag("packagesScrollableList"),
-            horizontalArrangement = Arrangement.spacedBy(16.dp),
-            contentPadding = PaddingValues(top = 30.dp, start = 8.dp, end = 8.dp),
+            horizontalArrangement = Arrangement.spacedBy(20.dp), // Adjusted for spacing
+            contentPadding =
+                PaddingValues(top = 40.dp, start = 12.dp, end = 12.dp), // Increased padding
         ) {
           items(packages.size) { index ->
             // If package is selected, we display it bigger
             val isSelected = selectedIndex == index
-            val size by animateDpAsState(targetValue = if (isSelected) 335.dp else 320.dp)
+            val size by animateDpAsState(targetValue = if (isSelected) 350.dp else 320.dp)
 
             PackageCard(
                 packageProposal = packages[index],
                 isSelected = isSelected,
                 modifier =
-                    Modifier.width(250.dp)
+                    Modifier.width(260.dp) // Slightly wider for better touch targets
```

Comment:
The changes look great I was wondering if it would be nice to a width size increase as well on selected